rnote-ui/src/workspacebrowser/workspace_action/create_dir.rs

```diff
@@ -0,0 +1,74 @@
+use std::path::PathBuf;
+
+use gettextrs::gettext;
+use gtk4::{
+    gio, glib,
+    glib::clone,
+    pango,
+    traits::{BoxExt, ButtonExt, EditableExt, PopoverExt, StyleContextExt, WidgetExt},
+    Align, Button, Entry, Label, Popover, prelude::FileExt,
+};
+
+use crate::{WorkspaceBrowser, workspacebrowser::widget_helper};
+
+impl WorkspaceBrowser {
+    pub fn create_dir(&self) -> gio::SimpleAction {
```

Comment:
s/create_dir/create_dir_action

Reply:
I changed it to
```rust
    fn setup_dir_actions(&self, _appwindow: &RnoteAppWindow) {
        self.insert_action_group("workspace_action", Some(&self.imp().workspace_actions));

        self.imp().workspace_actions.add_action(&workspace_action::create_dir(self));
    }
```
like in the filerow-action.